src/modules/auth.js

```diff
@@ -0,0 +1,9 @@
+import JtockAuth from "j-tockauth";
+
+const auth = new JtockAuth({
+  host: "http://localhost:3000",
+  //host: process.env.REACT_APP_BASEURL,
```

Comment:
Remove, unless we want to take this opportunity to introduce those environment variables

Reply:
done